RELEASE.md

```diff
@@ -78,8 +75,8 @@ File the following CLOUDP tickets for deploying the new release (you may need to
 - "Release Database Tools X.Y.Z to CM/OM" with a component of "Cloud Manager Upgrade" and assigned team of "Automation"
 - "Release Database Tools X.Y.Z to Atlas" with a component of "Atlas Upgrade" and assigned team of "Atlas Triage"
 
-#### Close Release Ticket
-Move the JIRA ticket tracking this release to the "Closed" state.
```

Comment:
The status of this ticket gets changed to "In Code Review" once you open the changelog PR, so I moved this step to the end.